packages/docs/docs/circuits-api/web-prover.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Web Prover
+---
+
+The circuits package includes a browser compatible prover. This prover loads the proving keys from a remote URL. By default this url is `https://keys.unirep.io/2-beta-3/`.
+
+The server is expected to serve the `zkey`, `wasm`, and `vkey` files at their respective names in the provided subpath. e.g. for the above url the signup zkey is at `https://keys.unirep.io/2-beta-3/signup.zkey`.
```

Comment:
It could also be
`https://keys.unirep.io/${version}/`

and for example
the signup zkey of version 2-beta-3 is `https://keys.unirep.io/2-beta-3/signup.zkey`